docs/source/user/getting_started/advanced_workflow_concepts.rst

```diff
@@ -0,0 +1,309 @@
+.. jinja:: first_ctx
+
+Advanced workflow concepts
+###########################
+
+Resources
+----------
+
+Requesting resources can be done using a ``resources`` block, either for the whole workflow at the top level,
+
+.. code-block:: yaml
+
+    resources:
+    any:
+      scheduler: sge # Setting the scheduler is not normally needed because a
+                     # `default_scheduler` will be set in the config file.
+      scheduler_args:
+        shebang_args: --login
+        options:
+          -l: short
+
```

Comment:
This needs to be updated as per https://github.com/hpcflow/hpcflow-new/pull/805. So:

```suggestion
      shell_args:
        executable_args: ["--login"]
      scheduler_args:
        directives:
          -l: short
```